web/client/epics/__tests__/contextmanager-test.js

```diff
@@ -57,16 +62,18 @@ describe('contextmanager epics', () => {
             done();
         }, {});
     });
-    it('reload on contextSaved', (done) => {
-        const startActions = [contextSaved("Search Text")];
-        testEpic(reloadOnContexts, 1, startActions, ([a]) => {
-            expect(a.type).toBe(SEARCH_CONTEXTS);
-            expect(a.options).toExist();
-            expect(a.options.params).toExist();
-            expect(a.options.params.start).toBe(0);
-            expect(a.options.params.limit).toBe(12);
-            expect(a.text).toBe("test");
-            done();
+    it('reload on contextSaved, loginSuccess, logout', (done) => {
+        const startActions = [contextSaved("Search Text"), loginSuccess(), logout()];
+        testEpic(reloadOnContexts, 3, startActions, (actions) => {
+            actions.forEach(a => {
+                expect(a.type).toBe(SEARCH_CONTEXTS);
+                expect(a.options).toExist();
+                expect(a.options.params).toExist();
+                expect(a.options.params.start).toBe(0);
+                expect(a.options.params.limit).toBe(12);
+                expect(a.text).toBe("test");
+                done();
```

Comment:
This test stops at first iteration calling done. But you're expecting 3 actions.Maybe you have to test event the second and the 3rd action